Running `pytest` on freshly cloned repo results in errors

In trying to address #8 , I installed pytest via `conda install pytest` and encountered the following: 

```python
=========================================================== test session starts ===========================================================
platform darwin -- Python 3.7.10, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /Users/chananas2/Documents/madbytetesting/madbyte
collected 14 items

tests/test_madbyte_filter.py ...........                                                                                            [ 78%]
tests/test_madbyte_utils.py FF.                                                                                                     [100%]

================================================================ FAILURES =================================================================
______________________________________________________ test_establish_file_structure ______________________________________________________

project_dir = PosixPath('/private/var/folders/ws/4b3kt9f12jn3jvlp50f_x_ljl4jcfb/T/pytest-of-chananas2/pytest-3/data0')

    def test_establish_file_structure(project_dir: Path):
        out_dir = project_dir.joinpath("test")
>       utils.establish_file_structure("test", out_dir, project_dir)
E       TypeError: establish_file_structure() takes 1 positional argument but 3 were given

tests/test_madbyte_utils.py:11: TypeError
_____________________________________________________________ test_get_master _____________________________________________________________

project_dir = PosixPath('/private/var/folders/ws/4b3kt9f12jn3jvlp50f_x_ljl4jcfb/T/pytest-of-chananas2/pytest-3/data0')

    def test_get_master(project_dir: Path):
>       df = utils.get_master(project_dir)

tests/test_madbyte_utils.py:18:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
madbyte/utils.py:631: in get_master
    return pd.read_json(master_path)
/usr/local/Caskroom/miniconda/base/envs/madbyte/lib/python3.7/site-packages/pandas/util/_decorators.py:199: in wrapper
    return func(*args, **kwargs)
/usr/local/Caskroom/miniconda/base/envs/madbyte/lib/python3.7/site-packages/pandas/util/_decorators.py:299: in wrapper
    return func(*args, **kwargs)
/usr/local/Caskroom/miniconda/base/envs/madbyte/lib/python3.7/site-packages/pandas/io/json/_json.py:563: in read_json
    return json_reader.read()
/usr/local/Caskroom/miniconda/base/envs/madbyte/lib/python3.7/site-packages/pandas/io/json/_json.py:694: in read
    obj = self._get_object_parser(self.data)
/usr/local/Caskroom/miniconda/base/envs/madbyte/lib/python3.7/site-packages/pandas/io/json/_json.py:716: in _get_object_parser
    obj = FrameParser(json, **kwargs).parse()
/usr/local/Caskroom/miniconda/base/envs/madbyte/lib/python3.7/site-packages/pandas/io/json/_json.py:831: in parse
    self._parse_no_numpy()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <pandas.io.json._json.FrameParser object at 0x7fa785a90dd0>

    def _parse_no_numpy(self):

        json = self.json
        orient = self.orient

        if orient == "columns":
            self.obj = DataFrame(
>               loads(json, precise_float=self.precise_float), dtype=None
E               ValueError: Expected object or value

/usr/local/Caskroom/miniconda/base/envs/madbyte/lib/python3.7/site-packages/pandas/io/json/_json.py:1079: ValueError
========================================================= short test summary info =========================================================
FAILED tests/test_madbyte_utils.py::test_establish_file_structure - TypeError: establish_file_structure() takes 1 positional argument bu...
FAILED tests/test_madbyte_utils.py::test_get_master - ValueError: Expected object or value
====================================================== 2 failed, 12 passed in 1.31s =======================================================

~/Documents/madbytetesting/madbyte main
```

--- a/madbyte/core.py
+++ b/madbyte/core.py
@@ -7,7 +7,7 @@ import pandas as pd
 import madbyte.utils as utils
 from madbyte.filter import Filter, SOLVENT_DICT
 from madbyte.logging import get_logger
-from madbyte.parsers import acd_sim_parser, mestrenova_parser, topspin_parser
+from madbyte.parsers import acd_sim_parser, mestrenova_parser, topspin_parser,csv_parser
 
 logger = get_logger("MADByTE")
 
@@ -51,6 +51,9 @@ def construct_spin_system(
             hsqc_data, tocsy_data = topspin_parser(name, input_dir, output_dir)
         elif nmr_data_type == "Mestrenova":
             hsqc_data, tocsy_data = mestrenova_parser(name,input_dir,output_dir)
+        elif nmr_data_type == "CSV":
+            hsqc_data, tocsy_data = csv_parser(name,input_dir,output_dir)
+            merge_multiplets=False
         else:
             raise Exception("nmr_data_type not valid!")
 

--- a/madbyte/parsers.py
+++ b/madbyte/parsers.py
@@ -244,3 +244,22 @@ def mestrenova_parser(name, input_dir, output_dir, tocsy_precision=3, hsqc_preci
     )
 
     return hsqc_data, tocsy_data
+
+def csv_parser(name, input_dir, output_dir, tocsy_precision=3, hsqc_precision=3):
+    #Allows for manual input of data as CSV files # 
+    Data_Path = os.path.join(input_dir,name)
+    for data in os.listdir(Data_Path):
+        if 'HSQC' in data: 
+            hsqc_data = pd.read_csv(os.path.join(Data_Path,data))
+            hsqc_data = hsqc_data.copy().astype("float").round(hsqc_precision).sort_values(by=["H_PPM"], ascending=True)
+            hsqc_data.Identity = name
+            hsqc_data.to_json(os.path.join(output_dir, f"{name}_HSQC_Preprocessed.json"),orient="records")
+        if 'TOCSY' in data: 
+            tocsy_data = pd.read_csv(os.path.join(Data_Path,data))
+            tocsy_data = tocsy_data.copy().astype("float").round(tocsy_precision).sort_values(by=["Ha"], ascending=True)
+            ### TOCSY IMPORT finishing steps
+            tocsy_data.Identity = name
+            tocsy_data.to_json(os.path.join(output_dir, f"{name}_TOCSY_Preprocessed.json"),orient="records")
+        else:
+            pass
+    return hsqc_data,tocsy_data

--- a/madbyte/plotting.py
+++ b/madbyte/plotting.py
@@ -123,4 +123,5 @@ def Bioactivity_plot(MasterOutput,Network_In_Path,Bioactivity_Data_In,title='Bio
         else:
             pass
     nx.set_node_attributes(Network_In,color_map,'_color')
+    nx.write_graphml(Network_In,str(title+'.graphml'))
     create_bokeh(Network_In, title, os.path.join(MasterOutput,fname+'.html'), extract_size=15, feature_size=10)

--- a/madbyte/utils.py
+++ b/madbyte/utils.py
@@ -535,7 +535,7 @@ def association_network(
     extract_edges = [(row.Spin_System_ID, row.Found_In) for row in master.itertuples()]
     spinsystem_edges = [
         (x,y,get_weight(x,y)) for x,y in combinations(idxs, r=2)
-        if corr_mat.loc[x,y] >= cutoff and not same_sample(x,y)
+        if get_weight(x, y) >= cutoff and not same_sample(x,y)
     ]
 
     # network building
@@ -548,20 +548,23 @@ def association_network(
     plot.create_bokeh(G, "MADByTE Full Association Network - All Spin Systems", project_dir.joinpath(f"{fname}_association_network_all.html"),
         extract_size=extract_size,feature_size=feature_size,)
     nx.write_graphml(G, project_dir.joinpath(f"{fname}_association_network_all.graphml"))
-
-    # Filters unconnected
-    H = trim_associations(G)
-    # nx.write_graphml(G, project_dir.joinpath("test_2.graphml"))
-    plot.create_bokeh(H, "MADByTE Similarity Network - Trimmed Spin Systems", project_dir.joinpath(f"{fname}_similarity_network.html"),
-        extract_size=extract_size,feature_size=feature_size,)
-    nx.write_graphml(H, project_dir.joinpath(f"{fname}_similarity_network_network.graphml"))
-
-    # Join connected associations
-    J = hybridize_network(H,idx_master, colors, hppm_error, cppm_error)
-    plot.create_bokeh(J, "MADByTE Hybrid Association Network", project_dir.joinpath(f"{fname}_hybrid_network.html"),
-        extract_size=extract_size,feature_size=feature_size,)
-    nx.write_graphml(J, project_dir.joinpath(f"{fname}_hybrid_network.graphml"))
-
+    try: 
+        # Filters unconnected
+        H = trim_associations(G)
+        # nx.write_graphml(G, project_dir.joinpath("test_2.graphml"))
+        plot.create_bokeh(H, "MADByTE Similarity Network - Trimmed Spin Systems", project_dir.joinpath(f"{fname}_similarity_network.html"),
+            extract_size=extract_size,feature_size=feature_size,)
+        nx.write_graphml(H, project_dir.joinpath(f"{fname}_similarity_network_network.graphml"))
+    except: 
+        print('Cannot Generate Similarity Network. \n If more than one sample was run, no similarities were found. \n If only was sample was run, please increase the number of samples to compare to in order to generate a similarity network.')
+    try: 
+        # Join connected associations
+        J = hybridize_network(H,idx_master, colors, hppm_error, cppm_error)
+        plot.create_bokeh(J, "MADByTE Hybrid Association Network", project_dir.joinpath(f"{fname}_hybrid_network.html"),
+            extract_size=extract_size,feature_size=feature_size,)
+        nx.write_graphml(J, project_dir.joinpath(f"{fname}_hybrid_network.graphml"))
+    except: 
+        print('Cannot Generate Hybrid Network.')
 
 def neighbour_spins(G, n, samples):
     return list(filter(lambda x: x not in samples, G.neighbors(n)))

--- a/madbyte_gui.py
+++ b/madbyte_gui.py
@@ -30,7 +30,7 @@ Dereplication_Database = 'Dereplication_Database'
 
 class MADByTE_Main(QMainWindow):
     def __init__(self):
-        __version__ = '1.2.0'
+        __version__ = '1.3.0'
         super(MADByTE_Main, self).__init__()
         uic.loadUi(os.path.join(BASE, 'static','MADByTE_GUI.ui'),self)
 
@@ -123,9 +123,15 @@ class MADByTE_Main(QMainWindow):
 
     ###Functions####
     def Launch_Documentation(self):
-        subprocess.Popen([os.path.join('Documentation','MADByTE_User_Manual.pdf')],shell=True)
+        try: 
+            subprocess.call(["open", os.path.join('Documentation','MADByTE_User_Manual.pdf')])
+        except: 
+            subprocess.Popen([os.path.join('Documentation','MADByTE_User_Manual.pdf')],shell=True)
     def Launch_Example(self):
-        subprocess.Popen([os.path.join('Documentation','MADByTE_Quick_Start_Tutorial.pdf')],shell=True)
+        try: 
+            subprocess.call(["open", os.path.join('Documentation','MADByTE_Quick_Start_Tutorial.pdf')])
+        except:
+            subprocess.Popen([os.path.join('Documentation','MADByTE_Quick_Start_Tutorial.pdf')],shell=True)
 
     def Load_Existing_Networks(self,MasterOutput):
         try:
@@ -650,6 +656,7 @@ if __name__ == "__main__":
     print("MADByTE is loading...")
     import sys
     app = QApplication(sys.argv)
+    app.setStyle('Windows')
     window = MADByTE_Main()
     window.setWindowIcon(QIcon(LOGO_PATH))
     if os.name == "nt":

--- a/madbyte_gui.py
+++ b/madbyte_gui.py
@@ -63,7 +63,7 @@ class MADByTE_Main(QMainWindow):
         self.MADByTE_Button_2.setEnabled(False)
         self.TOCSY_Net_Button_2.setEnabled(False)
         self.Multiplet_Merger_Checkbox.setChecked(True)
-        for NMR_Datatype in ['Bruker','Mestrenova']:#,'JOEL','Agilent','NMRPipe','Peak Lists]:
+        for NMR_Datatype in ['Bruker','Mestrenova','CSV']:#,'JOEL','Agilent','NMRPipe','Peak Lists]:
             self.NMR_Data_Type_Combo_Box.addItem(NMR_Datatype)
         self.Network_Filename_Input.setText("MADByTE")
         ### Bioactivity Layering values ###

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -3,12 +3,15 @@ import pandas as pd
 
 import pytest
 
+ROOTDIR = Path(__file__).parent.parent
+
 
 @pytest.fixture(scope="session")
 def project_dir(tmpdir_factory):
     fn = tmpdir_factory.mktemp("data")
+    print(fn)
     return Path(str(fn))
 
 @pytest.fixture
 def sample_data_dir():
-    return Path("./tests/").absolute().joinpath("sample_data")
+    return ROOTDIR / "tests" / "sample_data"

--- a/tests/test_madbyte_utils.py
+++ b/tests/test_madbyte_utils.py
@@ -7,10 +7,8 @@ from madbyte import utils
 
 
 def test_establish_file_structure(project_dir: Path):
-    out_dir = project_dir.joinpath("test")
-    utils.establish_file_structure("test", out_dir, project_dir)
+    utils.establish_file_structure(project_dir)
     assert project_dir.exists()
-    assert out_dir.exists()
     assert project_dir.joinpath("Spin_Systems_Master.json").exists()
 
 

